### DISCUSSION TITLE
Threat feeds

### DISCUSSION THREAD

**Discussion by SanibelJack** [Security-Onion-Solutions/securityonion#4633] [Unsupported Versions]
Question for the community; Is it possible for SO to ingest threat feeds to help with detection? 

Thank you

**Comment by 0nnyx** [comment]
Have a look at Wes github for Alienvault & MISP threat feed integration within Zeek:
https://github.com/weslambert/securityonion-misp
https://github.com/weslambert/securityonion-otx

**Comment by SanibelJack** [comment]
Thank you

Jack B. Voth

Sent from my iPhone.  Please excuse the brevity, spelling and punctuation.

On Jul 3, 2021, at 3:04 AM, 0nnyx ***@***.***> wrote:

﻿
Have a look at Wes github for Alienvault & MISP threat feed integration within Zeek:https://github.com/weslambert/securityonion-misphttps://github.com/weslambert/securityonion-otx —You are receiving t
Neutral (From: ***@***.******@***.***>, External)
  Report This Email<https://shared.outlook.inky.com/report?id=c2FuaWJlbGN5YmVydGVjaC9qYWNrLnZvdGhAc2FuaWJlbGN5YmVydGVjaC5jb20vMmQzNGE3NTY4NzQ1YmNiMzcxNmEzZDAyMWE0ZjE2NTAvMTYyNTI5NTg4NC45MQ==#key=4096e246f80307c6eac9a7ebc98041b4>  FAQ<https://www.inky.com/banner-faq/>  Protection by INKY<https://www.inky.com>


Have a look at Wes github for Alienvault & MISP threat feed integration within Zeek:
https://github.com/weslambert/securityonion-misp<https://shared.outlook.inky.com/link?domain=github.com&t=eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eJxtUN1OgzAUfpde45gTLiRZMjKRaKRmCY5ASJZSzkaxLbMtKJi9u8Ubo_H2nO__ExkggtUoQJpIVgGnYwXKAG2Qg5g0oCThKDgSrsFBYHGyM-zIKDGsk3pzYqbpqwXthMXPMr8OvbJc1Bhz1kHplu7Ps3TfQXMiZrPS1UB7xczYSSt6JZg-W_LzCwpkz7mDTKOAmAOHAaze0gabnejqvifZI69i7OdiP9bxviXb27c8w-pJFkMdN-H_GL-tVsshEbsJx9ENTnMPT7vrXGCWTPQDi2gq7sIxyaJl0UYrnIZDkuYjTh98nJ48vPX8ZLdez_00qAMIwviB1LUCrW2ultDXxdCZZvN30u9VLl8xUYDS.MEUCIArV0nkjZQqEmaHWJKbkamw9MWZMjWkbKeKXUoWNKvi-AiEA9LuPeftaLwGiqp-92DoAdCfljR36CabmjWUzO9nXHOA>
https://github.com/weslambert/securityonion-otx<https://shared.outlook.inky.com/link?domain=github.com&t=eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eJxtUN1OgzAUfpdezzHnuJBkychEopGaJbiFhWQp5WwU26LtAQdm727xxmi8Pef7_yQITImSBMQyLQqQvC_AIPCKTIjQCEYzSYIjkxYmBBxONyiOgjMUjbark8CqLaa8UQ4_yvw6tMZxSYX4ZoPcy72fZ-59gJVMjWa5Z4G3RmDfaCd61eDZcZ9fSKBbKScEKwMMDxI6cHIzl2s04vP7lu0eZRFTP1Pbvoy3NVvfvmc7ap70vivjKvwf49fFfNYlajPQOLqhabagw-Y6U1QkAz9TFQ37u7BPdtFsX0dzmoZdkmY9TR98mp4WdL3wk81yOdazYA6gmJAHVpYGrHW5asZfp12D1ervot-jXL4AqgCAdA.MEQCIBk5z-AsQ9E1hgL2rzIQbnh2jxy8dMNpQpKVGQXytzbgAiA2EocGoFb4OxKqQZb0IgHdm4mUidIhkwfU9LqQdYO9AQ>
